core/lib/spree/testing_support/common_rake.rb

```diff
@@ -32,6 +32,6 @@
   task :seed do |_t, _args|
     puts "Seeding ..."
 
-    sh "bundle exec rake db:seed RAILS_ENV=test"
+    sh "rake db:seed RAILS_ENV=test"
```

Comment:
I don't think we should use `bundle exec` here because we're already in ruby, and presumably already in a `bundle exec`. I'll remove this commit and submit it separately for discussion.